examples/graupnerbrunelstdp/stdputil.py

```diff
@@ -15,6 +15,22 @@
 
 logging.basicConfig(level=logging.WARN)
 
+#Note: having debug logging statements increases the run time by ~ 25%,
+# because they exist in tight loops, and expand their outputs, even when
+# debug is off, so we disable logging if possible.  Set this to true if
```

Comment:
No, that's not the take away at all.  It's more that we have to be careful about functions that are called many, many times (like logging.debug() in `time_above_threshold`)